src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -171,14 +171,29 @@ bool same_host_mem(cldnn::memory::ptr memPtr, uint8_t* hostPtr) {
     }
     return bufferMem == hostPtr;
 }
-
 }  // namespace
 
 namespace ov {
 namespace runtime {
 namespace intel_gpu {
 
+std::shared_ptr<const ov::Node> InferRequest::inputNodeByName(const std::string& name) {
+    std::shared_ptr<const ov::Node> resultNode = nullptr;
+    for (const auto& item : GetInputs()) {
+        if (item->get_friendly_name() == name)
+            resultNode = item;
+    }
+    return resultNode;
```

Comment:
Look like we can use the next code
```suggestion
    for (const auto& item : GetInputs()) {
        if (item->get_friendly_name() == name)
            rreturn item;
    }
    return nullptr;
```

Reply:
Fixed